modules/importer/src/server/mod.rs

```diff
@@ -155,6 +157,37 @@ impl Server {
 
         Ok(Some(result))
     }
+
+    /// Reset all jobs back into non-running state.
```

Comment:
Are we sure this is the behavior we want? Personally, I'd expect jobs to "pick up where they left off" or, if that's not possible, "start over from the beginning" if the server bounces. These import jobs are so long-running that I think we can't assume the server will stay up the whole time. What benefit is it to the user to force them to reconfigure them when that happens?

Reply:
Nothing needs to be reconfigured. Maybe the comment can be rephrased. But what it does is, it resets the state to "not running". Keeping all the rest.

It's not possible to pick up the work, as that state was lost due to the shutdown. Keeping the state would let the processor think the job is still running.

Ultimately, what we would need is a more sophisticated scheduler. Which splits the work into smaller pieces, reports back with the progress, checks for cancellation, allows for graceful shutdowns, and distributed work.

But I think that small improvement should be easy and fixes an immediate issue.